File: output/StarCoder/avatar/Python/Java/codeforces_546_A.java
import java.util.*;
public class codeforces_546_A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int k = sc.nextInt();
        int n = sc.nextInt();
        int w = sc.nextInt();
        if (1 <= k && k <= 1000 && 1 <= w && w <= 1000) {
            int total = 0;
            for (int i = 1; i <= w; i++) {
                total += i * k;
            }
            if (total > n) {
                System.out.println(total - n);
            } else {
                System.out.println(0);
            }
        }
    }
}

File: output/CodeGen/avatar/Java/Python/codeforces_546_A.py
import java.util.Scanner ; public class codeforces_546_A { public static void main ( String [ ] args ) { Scanner sc = new Scanner ( System.in ) ; long pandu, vundu, urdu, c = 0 ; pandu = sc.nextLong ( ) ; vundu = sc.nextLong ( ) ; urdu = sc.nextLong ( ) ; for i in range ( 1, urdu + 1 ) : c += i * pandu ; if c < vundu : print ( "0" ) ; else : print ( c - vundu ) ; } }